src/utils/network-monitor.ts

```diff
@@ -399,21 +351,15 @@ export class NetworkMonitor {
     const output: {[key: string]: ProviderStatus} = {}
 
     for (const network of supportedNetworks) {
-      if (
-        !(network in this.configFile.networks) ||
-        !('providerUrl' in this.configFile.networks[network]) ||
-        this.configFile.networks[network].providerUrl === undefined ||
-        this.configFile.networks[network].providerUrl === ''
-      ) {
-        output[network] = ProviderStatus.NOT_CONFIGURED
+      const networkConfig = this.configFile.networks[network]
+      if (networkConfig?.providerUrl) {
+        output[network] = this.providers[network]
+          ? this.ws[network]?.readyState === WebSocket.OPEN
+            ? ProviderStatus.CONNECTED
+            : ProviderStatus.DISCONNECTED
+          : ProviderStatus.DISCONNECTED
       } else {
-        output[network] = this.providers[network] === undefined ? ProviderStatus.DISCONNECTED : ProviderStatus.CONNECTED
-        // check if using a WebSocketProvider connection
-        if (output[network] === ProviderStatus.CONNECTED && network in this.ws) {
-          // using WebSocketProvider, do a more thorough test
-          output[network] =
-            this.ws[network].readyState === WebSocket.OPEN ? ProviderStatus.CONNECTED : ProviderStatus.DISCONNECTED
```

Comment:
@alexanderattar ok i think removing this is ok. We have moved away from the websocket provider.